docs(backups): extra details (#9693)
vatesfr/xen-orchestra: docs/docs/backups.md

That's it! Your chosen file will be restored.

> Alternatively, administrators can also restore files with the command-line interface, see [guide](https://github.com/vatesfr/xen-orchestra/blob/master/%40vates/fuse-vhd/README.md#restore-a-file-from-a-vhd-using-fuse-vhd-cli).

Reviewer: ```suggestion
Alternatively, administrators can also restore files with the command-line interface. See [this guide](https://github.com/vatesfr/xen-orchestra/blob/master/%40vates/fuse-vhd/README.md#restore-a-file-from-a-vhd-using-fuse-vhd-cli) to know more.
```

The start of the week is computed with the timezone set in the schedule.
- **What GFS isn't:**\
GFS in Xen Orchestra stands for Grandfather-Father-Son. It's a backup strategy, and is not related to the file system called GFS2 (or Global File System 2), supported by XenServer.

> - GFS retention is defined per schedule. For example, if a backup has two schedules, two independent GFS backups will be created. We therefore recommend having a single schedule per job that uses GFS long-term retention.

Reviewer: ```suggestion
- GFS retention is defined per schedule. For example, if a backup has two schedules, two independent GFS backups will be created. This is why we recommend using a single schedule for any job utilizing GFS long-term retention.
```
